packages/account/src/Styles/account.scss

```diff
@@ -213,6 +213,18 @@ $MIN_HEIGHT_FLOATING: calc(
                 }
             }
         }
+        @include mobile-or-tablet-screen {
+            &__financial-assessment {
+                max-width: 60rem;
+                margin: 0 auto;
+                .account-form__fieldset {
+                    max-width: 100%;
```

Comment:
Let's move this `max-width: 100%` and other common properties to `fieldset` property inside `account-common.scss` 🙏 Since, we do this for other sections also.